ambry-store/src/main/java/com/github/ambry/store/PersistentIndex.java

```diff
@@ -2303,11 +2303,17 @@ void setGetBlobReadInfoTestCallback(Runnable getBlobReadInfoTestCallback) {
    * @param logSegmentName the name of the log segment whose index segment files are required.
    * @return the list of {@link IndexSegment} files that refer to the log segment with name {@code logSegmentName}.
    */
-  static File[] getIndexSegmentFilesForLogSegment(String dataDir, final LogSegmentName logSegmentName) {
+  static File[] getIndexSegmentFilesForLogSegment(String dataDir, final LogSegmentName logSegmentName) { // ERROR
```

Comment:
also method `cleanupIndexSegmentFilesForLogSegment` has to be updated as well.

Reply:
Good catch.